### PR TITLE
Documentation clean up

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: docs/Home.md [hunk 1]
```diff
@@ -170,35 +170,34 @@ provided by the helm project for configuring helm/tiller on your GKE cluster.
...
+
 ## Usage
 
 ### Google Kubernetes Engine Build Step Configuration
 
 Each GKE Build Step configuration can point to a different GKE cluster. Follow the steps below to
 create one.
 
-#### GKE Build Step Parameters
+##### GKE Build Step Parameters
```
Comment (stephenashank) [inline docs/Home.md]: Nit: Why did you change the header level here?

Comment (averikitsch) [reply docs/Home.md]: It looked like a step in the build configuration to me, so I thought I'd try to have it distinguished from the other headers.

Comment (stephenashank) [reply docs/Home.md]: Makes sense, thanks.
